Content/Items/Utility/LaserPointer.cs

```diff
@@ -110,11 +112,14 @@ public override void SetDefaults()
 			Projectile.penetrate = -1;
 		}
 
-		public override void AI()
-		{
-			Projectile.Center = Owner.Center;
-			LaserRotation = (Main.MouseWorld - Owner.Center).ToRotation();
-			Owner.heldProj = Projectile.whoAmI;
+        public override void AI()
```

Comment:
Wahts up with the whitespace changing here? Have we not run the formatter on this?